content/spaces/setup.textile

```diff
@@ -0,0 +1,77 @@
+---
+title: SDK setup
+meta_description: "Install and instantiate the Spaces SDK."
+languages:
+  - javascript
+---
+
+Use these instructions to setup the Spaces SDK.
+
+h2(#api). Get an API key
+
+An "API key":/auth#api-keys is required to instantiate a client and authenticate with Ably. API keys are used either to "authenticate directly":/auth/basic, or to generate "Tokens":/auth/token for untrusted clients. "Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
+
+To use the Spaces SDK, the API key needs the following "capabilities:":/auth/capabilities
+
+* Publish
+* Subscribe
+* Presence
+* History
+
+h2(#install). Install and instantiate
```

Comment:
I've made some changes to this section in the GitHub readme just for your reference.